static/js/formatters-internal.js

```diff
@@ -294,7 +294,7 @@ export function image(simpleOrComplexImage = {}, size = '200x', atLeastAsLarge =
     return img;
```

Comment:
can we add jsdoc for the params of this image function?

Reply:
sure!